james947/Dunda
# Merge pull request #1 from james947/ch-restructure-application

#### What does this PR do?
* Enables user login 
#### Description of Task to be completed?
* Setup authentication using the Spotify API
#### How should this be manually tested?
* Run the Vue App 
* Run the Node server
* Navigate to the homepage on the Vue App
* Click on the button `Login with Spotify`
* Ensure that a token is returned in the URL

#### What are the relevant trello stories?
[Login](https://trello.com/c/wp8DcHtL/2-enable-user-login)
#### Any background context you want to add?
* N/A
#### Screenshots
* 
![image](https://user-images.githubusercontent.com/29922960/52353837-5db61b80-2a40-11e9-8248-cbc5cb3b6dca.png)
